src/java/com/unifina/signalpath/remote/Http.java

```diff
@@ -170,9 +189,19 @@ protected void sendOutput(HttpTransaction call) {
 		}
 	}
 
-	private static boolean isOctetStream(HttpEntity entity) {
+	private static String getDataType(HttpEntity entity) {
```

Comment:
Should we only work with standard content types (text/plain, application/xml, etc) here instead of labels (json, binary, etc)?